blocks/identity-block/components/HeadlinedSubmitForm/styles.scss

```diff
@@ -18,7 +18,7 @@
     }
 
     &:last-of-type {
-      margin-bottom: map-get($spacers, 'sm');
+      margin-bottom: map-get($spacers, 'md');
```

Comment:
```suggestion
      margin-bottom: map-get($spacers, 'sm');
```

Reply:
Last input (the confirm password) should be `sm` bottom margin, so this one should not have been changed.
![image](https://user-images.githubusercontent.com/2287238/133159573-3b84184f-d268-4da3-972f-fccdb938ed58.png)